.github/workflows/data-prepper-trace-analytics-raw-span-compatibility-e2e-tests.yml

```diff
@@ -24,5 +24,8 @@ jobs:
         java-version: ${{ matrix.java }}
     - name: Checkout Data-Prepper
       uses: actions/checkout@v2
-    - name: Run raw-span compatibility end-to-end tests with Gradle
-      run: ./gradlew :e2e-test:trace:rawSpanCompatibilityEndToEndTest
```

Comment:
I propose that we keep the `rawSpanCompatibilityEndToEndTest`. It can just be a task which depends on the other two tests.

This approach allows for developers to have few test commands to run locally.

Reply:
So the current rawSpanCompatibilityEndToEndTest is renamed as rawSpanOTLPLatestReleaseCompatibilityEndToEndTest. I only added a similar test with event type named rawSpanEventLatestReleaseCompatibilityEndToEndTest. Can you specify what two other tests this should depend on?